src/main/java/hw/hw11/Phonebook.java

```diff
@@ -0,0 +1,38 @@
+package hw.hw11;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Phonebook {
+    private List<Record> records;
+
+    public Phonebook() {
+        records = new ArrayList<>();
+    }
+
+
+    public void add(Record record) {
+        records.add(record);
+    }
+
+
+    public Record find(String name) {
+        for (Record record : records) {
+            if (record.getName().equals(name)) {
+                return record;
+            }
+        }
+        return null;
+    }
+
+
+    public List<Record> findAll(String name) {
+        List<Record> foundRecords = new ArrayList<>();
+        for (Record record : records) {
+            if (record.getName().equals(name)) {
+                foundRecords.add(record);
+            }
+        }
+        return foundRecords.isEmpty() ? null : foundRecords;
```

Comment:
```suggestion
        return foundRecords;
```
на уроці обговорювали - це погана практика повертати null для колекцій. 
краще вона буде пустою, ніж null